src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1094,25 +1094,33 @@ a.test-arrow:hover{
 	margin-right: 5px;
 }
 
-.sub-variant, .sub-variant > h3 {
-	margin-top: 0px !important;
-	padding-top: 1px;
+h3.variant {
+	font-weight: 600;
+	font-size: 1.1em;
+	margin-bottom: 10px;
+	border-bottom: none;
 }
 
-#main .sub-variant > h3 {
-	font-size: 15px;
-	margin-left: 25px;
-	margin-bottom: 5px;
+.sub-variant h4 {
+	font-size: 1em;
+	font-weight: 400;
+	border-bottom: none;
+	margin-top: 0;
+	margin-bottom: 0;
 }
 
-.sub-variant > div {
-	margin-left: 20px;
-	margin-bottom: 10px;
+.sub-variant {
+	margin-left: 24px;
+	margin-bottom: 20px;
```

Comment:
I feel like there's not quite enough visual separation between the fields' docs and the variant's docs:

<img src="https://user-images.githubusercontent.com/37223377/139512215-49ea2369-24b5-4544-a1a1-447395b76366.png" height="300">

Adding more space between them (e.g., `margin-bottom: 40px` here) could help, but I wonder if there's a better way to separate them. Do you have any ideas?

Reply:
I went with the `margin-bottom: 40px`.